src/Definition/PhpEnumType.php

```diff
@@ -22,35 +23,38 @@ class PhpEnumType extends EnumType
     public const MULTIPLE_DEPRECATIONS_DISALLOWED = 'Using more than 1 Deprecated attribute is not supported.';
 
     /**
-     * @var class-string<\UnitEnum>
+     * @var class-string<\BackedEnum>
      */
     protected string $enumClass;
 
     /**
-     * @param class-string<\UnitEnum> $enum
+     * @param class-string<Enum> $enumClass
```

Comment:
As far as I know, `\Enum` isn't a real thing? 🤔 It probably should be `\UnitEnum`